.github/workflows/ci.yml

```diff
@@ -112,7 +112,7 @@ jobs:
 
   build-ubuntu-latest-pcp:
     # Turns out 'ubuntu-latest' can be older than 20.04, we want PCP v5+
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Please note the comment there. Maybe update to `ubuntu-22.04` instead?

Reply:
`ubuntu-22.04` doesn't exist as Github virtual environment unitl now, maybe on August.

Reply:
Done: https://github.com/htop-dev/htop/pull/981/commits/79a329d1602fd47fd5431c2b2bd15f3d98d0e2c2